apps/meteor/ee/server/startup/deviceManagement.ts

```diff
@@ -5,7 +5,9 @@ import { onToggledFeature } from '../../app/license/server/license';
 onToggledFeature('device-management', {
 	up: () =>
 		Meteor.startup(async () => {
-			const { createPermissions } = await import('../lib/deviceManagement/startup');
+			const { createPermissions, createDeviceManagementModal } = await import('../lib/deviceManagement/startup');
 			await createPermissions();
+			await createDeviceManagementModal();
+			await import('../lib/deviceManagement/deviceManagementModal');
```

Comment:
Given that you're only importing this, does this file/endpoint needs to be a meteor method? Can't it be a normal endpoint? 👀

Reply:
For now, its easier to implement the front-end functionality with a meteor method, so it was requested by me to create a meteor method for this, will migrate in future.

Thanks for the quick help Kevin!